tools/ci/regen_certs.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import base64
+import logging
+import subprocess
+
+
+logging.basicConfig()
+logger = logging.getLogger()
+
+
+# TODO(Issue #24180): Regenerate SXG fingerprint too.
+CHROME_SPKI_CERTS_CONTENT = """\
+# This file is automatically generated by 'wpt regen-certs'
+# DO NOT EDIT MANUALLY.
+
+# tools/certs/web-platform.test.pem
+WPT_FINGERPRINT = '{}'
```

Comment:
Nit, maybe use named param here to avoid potential footgun

```suggestion
WPT_FINGERPRINT = '{wpt_fingerprint}'
```

Remember to change the format call to use kwargs, too.